packages/react/src/analytics/integrations/shared/GoogleConsentMode/types.ts

```diff
@@ -0,0 +1,34 @@
+import { type ConsentData } from '@farfetch/blackout-analytics';
+
+export enum GoogleConsentType {
+  Granted = 'granted',
+  Denied = 'denied',
+}
+
+export type GoogleConsentRegionConfig = {
```

Comment:
This type is not being used

```suggestion
export type GoogleConsentRegionConfig = {
```

Reply:
fixed